CHANGELOG.md

```diff
@@ -1,9 +1,28 @@
-# dbt_facebook_pages v0.UPDATE.UPDATE
+# dbt_facebook_pages v0.3.0
 
- ## Under the Hood:
+[PR #11](https://github.com/fivetran/dbt_facebook_pages/pull/11) includes the following breaking changes:
+## 🚨 Breaking Changes 🚨:
+- This change is made breaking since columns have been removed in the source package (see the [dbt_facebook_pages_source v0.3.0 CHANGELOG](https://github.com/fivetran/dbt_facebook_pages_source/blob/main/CHANGELOG.md#dbt_facebook_pages_source-v030) for more details). 
+    - No columns were removed from the end models in this package, however if you use the staging models independently, you will need to update your downstream use cases accordingly.
+    - Columns removed from staging model `stg_facebook_pages__daily_page_metrics_total`:
+        - `consumptions`
+        - `content_activity`
+        - `engaged_users`
+        - `places_checkin_mobile`
+        - `views_external_referrals`
+        - `views_logged_in_total`
+        - `views_logout`
+    - Columns removed from staging model `stg_facebook_pages__lifetime_post_metrics_total`:
+        - `impressions_fan_paid`
+
+## Features
```

Comment:
It doesn't look like schema documentation was updated in this package, so this can be removed.